src/main/java/org/opensearch/sdk/rest/BaseExtensionRestHandler.java

```diff
@@ -41,7 +43,9 @@
  */
 public abstract class BaseExtensionRestHandler implements ExtensionRestHandler {
 
-    private static String extensionName;
+    private static final String VALID_ROUTE_PREFIX_PATTERN = "^[a-zA-Z0-9:/*_]*$";
```

Comment:
Is the equivalent constant in OpenSearch `public` so we can use it here instead of duplicating it?

Reply:
That constant is `VALID_ACTION_PREFIX` defined in TransportService. This same pattern is defined in NamedRoute but it isn't public.